desktop/apps/artwork/components/current_auctions/current_auctions.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { PropTypes } from 'react'
+import { partition, take } from 'underscore'
+import { upcomingLabel } from '../../lib/auction_label'
+
+export default function CurrentAuctions ({ auctionContextId, sales }) {
```

Comment:
A meta-comment is that we'll want to use this component on the `/auction2` page as well, so this could be abstracted to a different subfolder, not necessarily under the `artwork` app. That could be done now or wait until that work item, just something to think about.